draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -430,60 +437,121 @@ token issuance servers.
-but it's important to point out that this general architecture doesn't prevent these attacks. It does, however, move the
-threat model to this kind of analysis rather than just looking at direct metadata.
+While straightforward violations of user privacy like this may seem straightforward to mitigate, it
+remains an open problem to determine whether a certain set of information reveals "too much" about a
+specific user. There is ample evidence of data being assumed "private" or "anonymous" but, in hindsight,
+winds up revealing too much information such that it allows one to link back to individual
+clients; see {{?DataSetReconstruction=DOI.10.1109/SP.2008.33}} and {{CensusReconstruction}}
+for more examples of this in the real world, and see{{security-considerations}} for more discussion.
+
+# Impacts of Partitioning
+
+Applying privacy partitioning to communication protocols lead to a substantial change in communication patterns.
+For example, instead of sending traffic directly to a service, essentially all user traffic is routed through
+a set of intermediaries, possibly adding more end-to-end round trips in the process (depending on the system
+and protocol). This has a number of practical implications, described below.
+
+1. Service operational or management challenges. Information that is traditionally passively observed in the
+   network or metadata that has been unintentionally revealed to the service provider cannot be used anymore
+   for e.g. existing security procedures such as application rate limiting or DDoS mitigation.
+   However, network management techniques deployed at present often rely on information that is exposed by
+   most traffic but without any guarantees that the information is accurate. Privacy partitioning provides
+   an opportunity for improvements in these management techniques by providing opportunities to actively
+   exchange information with each entity in a privacy-preserving way and requesting exactly the information
+   needed for a specific task or function rather then relying on assumption that are derived on a limited
+   set of unintentionally revealed information which cannot be guaranteed to be present and may disappear
+   any time in future.
+
+1. Varying performance effects. Depending on how context separation is done, privacy partitioning may
+   affect application performance. As an example, Privacy Pass introduces an entire end-to-end round
+   trip to issue a token before it can be redeemed, thereby decreasing perormance. In contrast, while
+   systems like CONNECT proxying may seem like they would regress performance, often times the highly
+   optimized nature of proxy-to-proxy paths leads to improved perforamnce. In general, while performance
+   and privacy tradeoffs are often cast as a zero sum game, in reality this is often not the case.
+   <!-- Do we want to same something about connection coalescing here as an example? -->
```

Comment:
Are you thinking of how it is harder to coalesce over a proxy, or easier?

Reply:
Ah, I see. Maybe remove this in the text and file an issue for us to discuss?

Reply:
Done!